.golangci.yml

```diff
@@ -13,7 +13,9 @@ linters-settings:
   errcheck:
     # path to a file containing a list of functions to exclude from checking
     # see https://github.com/kisielk/errcheck#excluding-functions for details
-    exclude: ./.errcheck-exclude
```

Comment:
would be nice to remove/update the comment (`path to a file containing a list of functions to exclude from checking`)

Reply:
Good catch! 😅